mscore/pluginCreator.cpp

```diff
@@ -430,15 +436,16 @@ void PluginCreator::load()
 //   savePlugin
 //---------------------------------------------------------
 
-void PluginCreator::savePlugin()
-      {
-      if (created) {
+void PluginCreator::doSavePlugin(bool saveas) 
+{
```

Comment:
Differs from the preferred banner style indent. Here and several times below too